src/App.js

```diff
@@ -17,13 +16,10 @@ class App extends Component {
 
   handleTimeChange(event) {
     this.setState({ time: event.target.value });
-  }
-
-  handleSubmit(event) {
-    event.preventDefault();
     this.setState({ result: this.state.distance * 1.60934 })
```

Comment:
I am just playing with the vs code github plugin...

Would be nice to have some test though